tests/string_segmenter_test.rb

```diff
@@ -2,7 +2,16 @@
 require 'string_segmenter'
 
 class StringSegmenterTest < Minitest::Test
-  # Your test methods go here.
-end
-
-# Remember, the names of all test methods should begin with "test_".
\ No newline at end of file
+  def test_strings_full_words
+  str = "avacadoburritocatdog"
+  assert_equal(["avacado", "burrito", "cat", "dog"], string_segmenter(str))
+  end
```

Comment:
Indentation example:

```
def test_strings_full_words
  str = "avacadoburritocatdog"
  assert_equal(["avacado", "burrito", "cat", "dog"], string_segmenter(str))
end
```

That's how it should be. See if you can properly indent the rest, based on this.